app/models/rating.rb

```diff
@@ -0,0 +1,4 @@
+class Rating < ApplicationRecord
+  belongs_to :dish
+  validates :score, inclusion: { in: 1..5 }
```

Comment:
Might be a good case for an enum if we want the numbers to have meaningful names: http://api.rubyonrails.org/classes/ActiveRecord/Enum.html